lib/ChangeDueDateDialog/LoanList.js

```diff
@@ -91,6 +92,7 @@ const LoanList = (props) => {
       formatter={{
         selected: loan => <Checkbox
           name={`selected-${loan.id}`}
+          aria-label={`select-${loan.id}`}
```

Comment:
Please talk to the PO and find out of `loan.id` is the best option here. Reading out a UUID won't be great; we might need a function/translation that concatenates instance-title, barcode, due-date, etc. Note that `aria-label` needs to be a string, not a component, so you'll need `useIntl` and `formatMessage()`.